samza-core/src/main/java/org/apache/samza/zk/ZkSessionMetrics.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.samza.zk;
+
+import org.apache.samza.metrics.Counter;
+import org.apache.samza.metrics.MetricsBase;
+import org.apache.samza.metrics.MetricsRegistry;
+
+public class ZkSessionMetrics extends MetricsBase {
+
+  /**
+   * Denotes the number of times the zookeeper client session had expired.
+   */
+  public final Counter zkSessionExpirations;
+
+  /**
+   *  Denotes the number of times the zookeeper client session disconnects had occurred.
```

Comment:
nit: remove "Denotes" .. consider removing comments in places where it's obvious from the name.

Reply:
Done.